lib/lotus/generators/action.rb

```diff
@@ -48,22 +48,23 @@ def start
           template_path: _template_path,
         }
 
+        case options[:test]
+        when 'rspec'
+          test_type = 'rspec'
+        else
+          test_type = 'minitest'
+        end
+
         templates = {
           'action.rb.tt' => _action_path,
-          'view.rb.tt'   => _view_path,
-          'template.tt'  => _template_path
+          'action_spec.' + test_type + '.tt' => _action_spec_path,
```

Comment:
I changed it anyway to string interpolation, it's easier to read and more elegant than concatenating the strings in the hash key